.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -248,20 +248,20 @@ jobs:
   upload-definitions:
     needs:
     - build-and-run
-    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    if: ${{ (fail() || success()) && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') }}
```

Comment:
`always()` runs on cancellations; I want this on failures or success (GH always adds a implicit `success()`, but only if no directive is used: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#status-check-functions